app/queues/dns/dns-queue.server.ts

```diff
@@ -18,8 +19,34 @@ declare global {
 function init() {
   logger.debug('DNS Queue and DNS Worker init');
 
+  const sendDnsRecordStatusUpdateNotification = async (
+    id: DnsRecord['id'],
+    status: DnsRecord['status']
+  ) => {
+    const dnsRecord = await getDnsRecordById(id);
+    if (!dnsRecord) {
+      throw new Error(`Could not get DNS Record with id=${id}`);
+    }
+    switch (status) {
+      case DnsRecordStatus.error:
+        logger.debug('adding job to send DNS record error notification');
```

Comment:
I would include info in the debug message from the `dnsRecord` (who owns it?  What is name, value, type?)